Frontend/src/Components/CreateListing/InfoSection.jsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 
 
-const InfoSection = ({ category, setCategory, brand, setBrand, condition, setCondition }) => {
+const InfoSection = ({ category, setCategory, brand, setBrand, condition, setCondition, price, setPrice}) => {
   return (
     <div className="infoSection">
       <h3>Info</h3>
```

Comment:
for lines 11-17, i'd recommend making the values (clothing, accessories, furniture, beauty, appliances) enums and using a map to render options for each enum. something like this https://stackoverflow.com/questions/58614249/use-enums-dynamically-for-option-tag-in-select-reactjs